doc/cosign_attest.md

```diff
@@ -41,8 +41,8 @@ cosign attest [flags]
 ```
       --allow-insecure-registry                                                                  whether to allow insecure connections to registries. Don't use this for anything but testing
       --attachment-tag-prefix [AttachmentTagPrefix]sha256-[TargetImageDigest].[AttachmentName]   optional custom prefix to use for attached image tags. Attachment images are tagged as: [AttachmentTagPrefix]sha256-[TargetImageDigest].[AttachmentName]
-      --cert string                                                                              path to the X.509 certificate in PEM format to include in the OCI Signature
-      --cert-chain string                                                                        path to a list of CA X.509 certificates in PEM format which will be needed when building the certificate chain for the signing certificate. Must start with the parent intermediate CA certificate of the signing certificate and end with the root certificate. Included in the OCI Signature
+      --certificate string                                                                       path to the X.509 certificate in PEM format to include in the OCI Signature
```

Comment:
should the help text reflect that these flags are aliased?

Reply:
We're using `RegisterAlias` for `cosign sign-blob` to alias `output` and `output-signature`. One downside is the help text shows each of these flags on separate lines, so it looks like they're two separate flags.

Besides that, aliasing and normalizing seem pretty similar. I prefer normalize because I ideally don't want to continue to support the old flag names - Maybe we can drop the normalize function in a 2.0 release?

Another benefit of normalize over aliasing is I don't have to alias the flag in each command, there's no GlobalRegisterAlias like with normalize.

Reply:
> One downside is the help text shows each of these flags on separate lines, so it looks like they're two separate flags.

Ugh 😭 

> I prefer normalize because I ideally don't want to continue to support the old flag names

That's a sufficient argument for me.